C-Plus-Plus/stack/Sort_stack_using_recursion.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+Problem Statement:
+ Given a stack, sort the given stack using recursion such that the greatest element is on the top. 
+
+Input: 
+The first line of input contains a single integer n-number of the element in the stack. The second line contains n space-separated integers: ith integer represent ith stack element.
```

Comment:
Modularize this line and make it more readable

Reply:
ok sir